src/bin/zmmailboxdctl

```diff
@@ -54,11 +54,6 @@ case "$1" in
   fi
 
   mkdir -p ${mailboxd_directory}/work/service/jsp
```

Comment:
What's in service jsp?

Reply:
IDK, some runtime data maybe. We will get rid of this in future. I have special note on this in SPIKE doc.